spec/features/tasks_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'feature_helper'
+
+describe "Tasks" do
```

Comment:
I think that would be a good idea to use `RSpec.feature "Tasks", type: :feature do`, what you think?